src/Symfony/Component/Messenger/Bridge/Amqp/Transport/Connection.php

```diff
@@ -443,6 +455,12 @@ private function setupExchangeAndQueues(): void
 
         foreach ($this->queuesOptions as $queueName => $queueConfig) {
             $this->queue($queueName)->declareQueue();
+            foreach ($queueConfig['bindings'] ?? [] as $binding) {
```

Comment:
In the example in the PR description, you show using an array key under `bindings` - like `manual-messages`, but that is not used here. What is the purpose of this YAML key? You also show an example without one. Which is the correct usage?